jobs/JGLOBAL_FORECAST

```diff
@@ -17,15 +17,23 @@ rCDUMP=${CDUMP}
 [[ ${CDUMP} = "gfs" ]] && export rCDUMP="gdas"
 
 # Forecast length for GFS forecast
-if [ ${CDUMP} = "gfs" ]; then
+case ${CDUMP} in
+  gfs | gefs)
```

Comment:
In the `gefs` tree, does `FHMAX_GFS` still make sense?
Should the line 23 be something like:
```
current_var=FHMAX_${CDUMP^^}; declare -x FHMAX=${current_var}
```
And so on.  
Beyond this point, there shouldn't be any references to `_gfs`, `_gefs` etc.

Reply:
Can, just makes the cases longer

Reply:
These kind of things are woven throughout the workflow, well beyond the little portion I am working on here.